src/PerformanceTests/NServiceBus7/App.config

```diff
@@ -6,19 +6,7 @@
   </configSections>
   
   <log4net>
-    <appender name="Console" type="log4net.Appender.ColoredConsoleAppender" >
-      <mapping>
-        <level value="ERROR" />
-        <foreColor value="Yellow" />
-      </mapping>
-      <mapping>
-        <level value="WARN" />
-        <foreColor value="Purple" />
-      </mapping>
-      <mapping>
-        <level value="FATAL" />
-        <foreColor value="RED" />
-      </mapping>
+    <appender name="Console" type="log4net.Appender.ConsoleAppender" >
```

Comment:
No colored console appender in .net core?

Reply:
Nope not supported. http://logging.apache.org/log4net/release/framework-support.html#appenders

Reply:
well, then we must live without any color :-)